source/deployment/sso-saml.rst

```diff
@@ -48,21 +40,21 @@ installed. If not, run - ``apt-get install libxmlsec1-openssl`` on
 Ubuntu - ``yum install xmlsec1-openssl`` on RHEL
 
```

Comment:
How about something like:

    Mattermost officially supports Okta and Microsoft ADFS as the identity providers (IDPs), 
    but you can also configure SAML for a custom IDP. (note absence of link to install doc, 
    and use of 'can' instead of 'may')

    - To set up SAML with Okta, see :doc:`sso-saml-okta`
    - To set up SAML with Microsoft ADFS, see :doc:`sso-saml-adfs`

Also, after looking at the Okta and ADFS doc, I see that each has its own Pre-installation section, so you can probably get rid of the Pre-installation section here. It's a duplication of effort, and can easily get out of sync with the Okta and ADFS SAML config docs.